services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -332,6 +333,8 @@ def _get_ports(
             {
                 "Protocol": "tcp",
                 "TargetPort": dynamic_sidecar_settings.DYNAMIC_SIDECAR_PORT,
+                "PublishedPort": random.randint(0, 50999),
+                "PublishMode": "host",
```

Comment:
something like `unused_port` should be used here

Reply:
done